test/reactTags.test.js

```diff
@@ -125,20 +125,21 @@ describe('Test ReactTags', () => {
     expect(actual).to.not.have.members(['Banana']);
   });
 
-  test('handles the paste event and splits the clipboard on delimiters', () => {
+  test('handles the paste event, splits the clipboard on delimiters, and ignores duplicates', () => {
```

Comment:
rename this to `should split the clipboard on delimiters` and remove duplicate tags in this test. Add a new `it` block `should not allow duplicate tags` and test for duplicate tags there.